docs/SPEC.md

```diff
@@ -3038,6 +3110,56 @@ If you need to convert other columns use the `map` function directly with the `u
 
 [IMPL#242](https://github.com/influxdata/platform/issues/242) Update specification around type conversion functions.
 
+#### Errors
+
+Errors selects error tables from the meta channel and places them on the data channel.
+The input data channel is dropped.
+The output meta channel is empty.
```

Comment:
Should an error be a specific type:

```
type error = {
     message string
     reference string
}
```

?

So that users know what to expect if they want to process errors?